docs/user-guide/generate-certificates.md

```diff
@@ -72,61 +109,100 @@ The following command output shows generation of a self signed PKCS12 certificat
 #>
 ```
 
-Because `--update-config` was specified the `zowe.certificates` section's values are updated to reference the newly generated certificate.  These updates are logged by the `zwe init certificate` command output.  Open the `zowe.yaml` file to check the references to the newly generated certificate values, as shown below:
+Values of the `zowe.certificates` section are updated with customization of `--update-config` to reference the newly generated certificate. These updates are logged by the `zwe init certificate` command output.
+
+Open the `zowe.yaml` file to check the references to the newly generated certificate values, as shown in the following code snippet:
+
+**Example YAML:**
 
 ```
   certificate:
     keystore:
       type: PKCS12
-      file: /global/zowe/keystore/localhost/localhost.keystore.p12
+      file: /var/zowe/keystore/localhost/localhost.keystore.p12
       password: password
       alias: localhost
     truststore:
       type: PKCS12
-      file: /global/zowe/keystore/localhost/localhost.truststore.p12
+      file: /var/zowe/keystore/localhost/localhost.truststore.p12
       password: password
     pem:
-      key: /global/zowe/keystore/localhost/localhost.key
-      certificate: /global/zowe/keystore/localhost/localhost.cer
-      certificateAuthorities: /global/zowe/keystore/local_ca/local_ca.cer
+      key: /var/zowe/keystore/localhost/localhost.key
+      certificate: /var/zowe/keystore/localhost/localhost.cer
+      certificateAuthorities: /var/zowe/keystore/local_ca/local_ca.cer
 ```
 
-When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept its untrusted certificate authority.  Depending on the browser you are using there are different ways to proceed.
+For details about the certificate you generated, run the following command:  
+ `keytool -v -list keystore localhost.keystore.p12`  
+For more information about additional commands to manage a keystore, see the [keytool documentation](https://docs.oracle.com/en/java/javase/11/tools/keytool.html).
+
+### Next steps after PKCS12 setup
 
-## Create a self signed JCERACFKS certificate
+When using a self-signed certificate, you will be challenged by your browser when logging in to Zowe to accept Zowe's untrusted certificate authority. Depending on the browser you are using there are different ways to proceed. See next steps about how to [import the PKCS12 certificate to your brower](./import-certificates#import-your-pkcs12-certificate).
 
-The following `zowe.yaml` example will generate:
+## Create a self-signed JCERACFKS certificate
 
- - A `JCERACFKS` certificate, specified in `zowe.setup.certificate.type` 
- - A keyring named `ZoweKeyring` specified in  `zowe.setup.certificate.keyring.name`. 
+Use the following procedure to configure zowe.yaml and create a self-signed PKCS12:
```

Comment:
Use the following procedure to configure zowe.yaml ~~and create a self-signed PKCS12~~: